README.rst

```diff
@@ -255,6 +257,57 @@ Details:
 - ``zcml-additional-fragments`` - Define additional zcml to load. See the `Additional ZCML`_ section.
 
 
+HaProxy / Supervisor integration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The `supervisor-haproxy`_ event listener tells haproxy to remove backends / add
+backends to the load balancing when supervisor detects instances to be stopping
+and starting.
+
+Example configuration:
+
+.. code:: ini
+
+    [buildout]
+    extends =
+        https://raw.githubusercontent.com/4teamwork/ftw-buildouts/master/production.cfg
+        https://raw.githubusercontent.com/4teamwork/ftw-buildouts/master/zeoclients/3.c
+        https://raw.githubusercontent.com/4teamwork/ftw-buildouts/master/haproxy.cfg
+
+    deployment-number = 05
+
+    # supervisor-haproxy-backend = plone${buildout:deployment-number}
+    # supervisor-haproxy-socket = tcp://localhost:8801
+
+
+The ``haproxy.cfg`` preconfigures the supervisor event listener when standardized
+names in HaProxy. If you do not follow our naming standards you may want to
```

Comment:
the `when standardized names in HaProxy` part of this sentence doesn't quite make sense to me.